mobile/lib/screens/kya/kya_lessons_page.dart

```diff
@@ -1,44 +1,57 @@
-import 'package:app/models/kya.dart';
+import 'package:app/screens/kya/kya_final_page.dart';
+import 'package:app/themes/app_theme.dart';
 import 'package:auto_size_text/auto_size_text.dart';
-import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_cache_manager/flutter_cache_manager.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:scrollable_positioned_list/scrollable_positioned_list.dart';
-
+import '../../models/enum_constants.dart';
```

Comment:
could we find an appropriate solution for these paths? So that we do not have to always do many ../../../... when importing modules? cc: @123MwanjeMike

Reply:
Alright @Mnoble-19 , I will let you collaborate with @123MwanjeMike on this. Thanks.